docs/guide/release.md

```diff
@@ -0,0 +1,42 @@
+# Release guidelines
+
+
+odfe-cli versions are expressed as **x.y.z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version, following [Semantic Versioning](https://semver.org/) terminology. 
+
+**The major, minor & patch version is related to odfe-cli compatibility, instead of odfe’s compatibility.**
+
+
+**Q: What goes into new patch version release train?**
+
+* Known bugs that prevent users from executing tasks. 
+    * eg: Failed to delete non-running detectors by delete command, failed to handle error.
+* Security vulnerabilities. 
+    * eg: leaking user credentials, insecure interaction with components.
+* cli’s dependency is updated with new patch version which fixes bugs and security issues that affects your feature. ( Treat your dependent library as your library)
```

Comment:
affect not affects

Not space before Treat

Reply:
Ack